config/puma.rb

```diff
@@ -0,0 +1,15 @@
+workers Integer(ENV["WEB_CONCURRENCY"] || 2)
+threads_count = Integer(ENV["MAX_THREADS"] || 5)
```

Comment:
Puma and or Unicorn are the recommended default web servers now, for Heroku and other places.